### DISCUSSION TITLE
404 Errors on static resources from the https://demo.django-wiki.org demo site

### DISCUSSION THREAD

**Discussion by yannlossouarn** [django-wiki/django-wiki#1339] [General]
Hi,

The demo site is failing, apparently due to all static resources not being found.

Bye,
Yann

**Comment by benjaoming** [comment]
Thanks for reporting! One of the fixtures for the demo has broken after django-nyt was upgraded. There's actually monitoring on the site, but seems it broke in a tricky way!

**Reply by benjaoming** [reply]
It's fixed, thanks so much for coming by and mentioning this. Judging from when django-nyt was released, the demo has been affected for 4-5 days.
